### PR TITLE
Create 2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md

### PR DESCRIPTION
**Pull request by dnbit**

### CODE REVIEW

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Flutter is currently in alpha version and open source with Google working actively on it. It provides a thin layer of C/C++ code and requires to use Dart for implementing the UI. All elements in Flutter are considered widgets that connect in a Matryoshka type hierarchy and uses Composition over Inheritance.
+For more info on the presentation check out the [slides](https://speakerdeck.com/passsy/flutter-60-fps-ui-of-the-future). Also, if you want to try an app build using Flutter, you can try Hamilton ([Android](https://play.google.com/store/apps/details?id=com.hamilton.app), [iOS](https://itunes.apple.com/us/app/hamilton-the-official-app/id1255231054?mt=8)).
+###Home Improvement: Architecture & Kotlin
+by Jorge D. Ortiz-Fuentes
+
+Being Android Architecture Components and Kotlin the two biggest announcements in the last Google I/O this was one of my top pics for day one. Interestingly, I had the chance to meet Jorge early in the morning before the opening keynote.
```
Comment (johsoe) [inline source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: picks*

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+During the talk, Jorge shared some insight of real and relevant scenarios where he explained how to improve our apps using Kotlin. Similarly to Florina Muntenescu’s talk, the room was not big enough, but this time I managed to get a sweet spot.
```
Comment (johsoe) [inline source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: Do you have enough points on the insights for a separate post?

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+I had the pleasure to meet Mario in my first Android meetup in London back in 2015. He is a Technology Evangelist in Amazon and former GDE. Therefore, I was looking forward to attending his talk and learn more about everything that Amazon is doing. 
+I was particularly impressed to see how things have evolved during the last years and how they have listened to the developers to make things easier with integrations that are incredibly simple and straight forward.
+###Reactive, Clean Architecture and Android Architecture Components, a Solid Formula for a MVVM Based Architecture
+by Lucia Payo
+
+My top pick for the day. Here again, I had the opportunity to meet Lucia before her talk as she was in N26 stand. We had the chance to discuss the new Android Architecture Components library and share different views on MVP and MVVM.
+In the talk, she presented a clean and well-structured reactive architecture based on MVVM. I would recommend having a look at her latest articles [here](https://mag.n26.com/reactive-clean-architecture-with-android-architecture-components-685a6682e0ca) and [here](https://mag.n26.com/practical-example-using-reactive-clean-architecture-approach-8a2436ea76b4).
```
Comment (johsoe) [inline source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: 👍  awesome with the links

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md (hunk 2 above)
Comment (dnbit) [reply source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: Not really, as I do not know Kotlin I cannot remember the specifics. Could do if the video was available but I can see that it has not been published yet.

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+### Keynote: Designing for Trust: User Experience + Privacy
+by Ame Elliott
+
+Ame is Design Director at [Simply Secure](https://simplysecure.org), a nonprofit organisation that provides professional education for user experience designers, researchers, and developers working on privacy, security, transparency, and ethics. She structured her talk into three different main topics: Understand the risks to users, Lead through design and Practical advice.
+Check the slides [here](https://simplysecure.org/resources//Elliott-DroidCon-UXandPrivacy.pdf).
+
+#### Developing a Streaming Android App for TV in Minutes
+by Mario Viviani
+
+I had the pleasure to meet Mario in my first Android meetup in London back in 2015. He is a Technology Evangelist in Amazon and former GDE. Therefore, I was looking forward to attending his talk and learn more about everything that Amazon is doing.
```
Comment (steffendsommer) [inline source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: at my first*
at Amazon*

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+### Day 2 Keynote
+by Ewa Macias, Behshad Behzadi, Mícheál Ó Foghlú
+
+Probably the most impressive part of the keynote was the demo on Artificial Intelligence and Google Assistant. We had the opportunity to see in action some of the most complex interactions with the Assistant that are still in the works including an incredible understanding in noisy environments.
+
+### Architecture Components
+by Florina Muntenescu
+
+Finally, I had the chance to attend Florina’s talk. There she talked about Architecture and best practice. She also introduced some of the latest additions to the Android Architecture Components library such as the new Paging library that is not yet released by the time I am writing this lines.
```
Comment (steffendsommer) [inline source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: these lines or this post *

**Comment by steffendsommer** [comment]
Does it make sense to include a sentence or two where you conclude the conference(s)? So readers looking for reviews can get your thoughts on it?

**Comment by johsoe** [comment]
Yep, a summing up at the end would make sense

**Comment by dnbit** [comment]
Fixed according to comments. Thanks for pointing out @steffendsommer

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md [hunk 6]
```diff
@@ -79,3 +79,9 @@ Training sessions during the event were completely packed, and some people found
 by Ben Weiss
 
 During his talk, Ben showed how to improve our app discoverability using Instant Apps where the key is modularising our apps. The primary constraint here seems to be the size of our modules which should be smaller than four megabytes. Also recommended tools to deal with this are the apk analyzer and configuration splits on top of the usual minification, shrinking and proguard obfuscation.
+
+# Final thoughts
+
+Droidcon Berlin was great to find out more about a wide range of topics related to Android and meet interesting people. GDD Krakow was a very big event where I have the opportunity to help in a training session, meet some of my mentees at Udacity and discuss different topics with Google Developer Advocates.
```
Comment (johsoe) [inline source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: *Meeting?

File: source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md [hunk 7]
```diff
@@ -79,3 +79,9 @@ Training sessions during the event were completely packed, and some people found
 by Ben Weiss
 
 During his talk, Ben showed how to improve our app discoverability using Instant Apps where the key is modularising our apps. The primary constraint here seems to be the size of our modules which should be smaller than four megabytes. Also recommended tools to deal with this are the apk analyzer and configuration splits on top of the usual minification, shrinking and proguard obfuscation.
+
+# Final thoughts
+
+Droidcon Berlin was great to find out more about a wide range of topics related to Android and meet interesting people. GDD Krakow was a very big event where I have the opportunity to help in a training session, meet some of my mentees at Udacity and discuss different topics with Google Developer Advocates. 
+
+Taking three flights for two conferences in a row in different countries is probably as tiring as it seems. Nevertheless, it was an awesome experience and I could not be happier about it.
```
Comment (johsoe) [inline source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md]: Maybe touch upon the talks as well? Best talk, most surprisingly interesting - that kind

**Pull request merged**